utils/objects/Conversion.py

```diff
@@ -0,0 +1,83 @@
+from dataclasses import dataclass
+
+
+NIFTI_TYPE = 'nii'
+NRRD_TYPE = 'nrrd'
+
+
+
+@dataclass
```

Comment:
I think most of this could also be done more clearly using a factory and or `properties`.  Happy to sync on it or discuss your thoughts